core/model/modx/modprocessor.class.php

```diff
@@ -947,6 +947,7 @@ class modObjectDuplicateProcessor extends modObjectProcessor {
     /** @var xPDOObject $newObject The newly duplicated object */
     public $newObject;
     public $nameField = 'name';
+    public $staticfileField = 'static_file';
     /** @var string $newNameField The name of field that used for filling new name of object.
```

Comment:
Shouldn't the changes being made here to the modObjectDuplicateProcessor be moved to a dedicated modStaticElementDuplicateProcessor? This seems like it could affect any object being duplicated.

Reply:
@opengeek It won't do anything if the object that is being duplicated is not static so I dont think this is really necessary